paver.sh

```diff
@@ -98,7 +98,13 @@ clean_build() {
     fi
     # In some case pip hangs with a build folder in temp and
     # will not continue until it is manually removed.
-    rm -rf /tmp/pip*
+    # On the OSX build server tmp is in $TMPDIR
+    if [ ! -z "${TMPDIR-}" ]; then
```

Comment:
Ah, true, I forgot about the draconian settings of `paver.sh`. This is because of `set -o nounset`. Parameter extensions is indeed the solution for that, I have also used it at https://github.com/chevah/python-package/blob/master/python-modules/chevah-python-test/get_binaries_deps.sh#L19

Interestingly the `-n` and `-z` don't work equally here… I looked at the manual and couldn't figure it out. But is seems as though `+` would be better than `-` here, from what I gather at https://www.gnu.org/software/bash/manual/bashref.html#Shell-Parameter-Expansion, as `+` doesn't substitute anything if the parameter is null or unset. However, in my tests, both work for this check in the bash shell included with OS X.